omero/sysadmins/unix/server-centos6-ice36.txt

```diff
@@ -129,8 +129,8 @@ Configure:
     :start-after: #end-release-ice36
     :end-before: #end-step04
 
-Installing OMERO.web
---------------------
+Installing and running OMERO.web
+--------------------------------
```

Comment:
This is correct since it depends on how you have installed it So only have the link to the web installation walkthrough